episodes/vaccine-comparisons.Rmd

```diff
@@ -0,0 +1,968 @@
+                                                 "vaccinated", by_group = FALSE)
+vaccinate_group_2_infections <- new_infections(output_vaccinate_group_2,
+                                               compartments_from_susceptible =
+                                                 "vaccinated", by_group = FALSE)
+vaccinate_group_3_infections <- new_infections(output_vaccinate_group_3,
+                                               compartments_from_susceptible =
+                                                 "vaccinated", by_group = FALSE)
+
+vaccinate_group_1_infections$intervention_type  <- "vaccinate group 1"
+vaccinate_group_2_infections$intervention_type  <- "vaccinate group 2"
+vaccinate_group_3_infections$intervention_type  <- "vaccinate group 3"
+
+output_infections <- rbind(baseline_infections,
+                           vaccinate_01_infections,
+                           vaccinate_group_1_infections,
+                           vaccinate_group_2_infections,
+                           vaccinate_group_3_infections)
+
+
+output_infections %>%
+  ggplot() +
+  geom_line(
+            aes(time, new_infections, colour = intervention_type,
+              linetype = intervention_type
+            ),
+            linewidth = 1) +
+  scale_y_continuous(
+    labels = scales::comma
+  ) +
+  geom_vline(
+    xintercept = c(
+      vaccinate_01$time_begin,
+      vaccinate_01$time_end
+    ),
+    linetype = 2
+  ) +
+  theme_bw() +
+  labs(
+    x = "Simulation time (days)",
+    y = "Individuals"
+  )
+```
+
+
+:::
+
+Vaccinating group 3 only results in the highest epidemic peak size. Targeting group 2 results in the smallest epidemic peak size. Whereas targeting all age groups pushes the epidemic peak time later.  
+
+## Vaccination versus NPIs
+
+Modelling is a useful tool for understanding the potential impact of the timing and different combinations of control measures. In an outbreak scenario, vaccines can be used alongside other control measures. A recent study by [Barnsley et al. (2024)](https://doi.org/10.1016/S2214-109X(24)00286-9) used modelling to show the impact the COVID-19 vaccine could have had alongside NPIs if the vaccine had been developed within 100 days of the pathogen threat being recognised. We will use this study as inspiration to show the impact of a vaccine versus NPI implemented at different times. 
+
+The NPI we will consider is closing schools. Specifically, a school closure that will reduce the contacts between school aged children (aged 0-15) by 0.5, and will cause a small reduction (0.01) in the contacts between adults (aged 15 and over). For the vaccine program, we assume an equal vaccination rate of 0.01 in each age group.
+
+We define **early implementation** as the first day of the simulation, and **late implementation** as 50 days after the start of the simulation. We assume the control measures are in place for 100 days after they start. 
```

Comment:
Maybe say 'first day of the simulation (i.e. when there are X infections in total)'? And similar for 50 days - as this is closer to the sort of triggers that would be used in real-life?